packages/frontend/src/components/staking/components/StakingFee.js

```diff
@@ -46,4 +46,15 @@ export default function StakingFee({ fee }) {
             <span className='fee'>{fee}%</span>
         </Container>
     );
+}
+
+export function StakingAPY({ apy }) {
```

Comment:
By convention, we export one component per file, as the default export, where the filename matches the class/function of the component we are exporting.  Please extract the `Container` into its own file and create a new file titled `StakingAPY.js` rather than export both a `default` and a `named` component from this file.